cmd/minikube/cmd/root.go

```diff
@@ -171,3 +179,16 @@ func ensureConfigFileExists(configPath string) {
 		}
 	}
 }
+
+func validateProxyArgs() {
+	if viper.IsSet(httpProxy) {
+		if ! util.ValidateProxyURI(viper.GetString(httpProxy)){
+			glog.Exitf("HTTP Proxy URL is not valid, Please check help message")
```

Comment:
Since this is a user feedback message, I think we should use ftm.println. Otherwise the user needs to deal with the "strange" log format